lib/app/handler/handler.go

```diff
@@ -530,15 +529,6 @@ func (h *WebHandler) exportApp(w http.ResponseWriter, req *http.Request,
 	if err := json.NewDecoder(req.Body).Decode(&config); err != nil {
 		return trace.Wrap(err)
 	}
-	_, _, err = net.SplitHostPort(config.RegistryHostPort)
```

Comment:
I don't believe this should do the validation.
Also, the changes to registry code seem to be allowing for optional HTTP scheme so this was wrong in enforcing the `host:port` form.